src/set_theory/game/basic.lean

```diff
@@ -105,15 +105,6 @@ theorem _root_.pgame.fuzzy_iff_game_fuzzy {x y : pgame} : pgame.fuzzy x y ↔ 
 instance covariant_class_add_le : covariant_class game game (+) (≤) :=
 ⟨by { rintro ⟨a⟩ ⟨b⟩ ⟨c⟩ h, exact @add_le_add_left _ _ _ _ b c h a }⟩
 
-instance covariant_class_swap_add_le : covariant_class game game (swap (+)) (≤) :=
-⟨by { rintro ⟨a⟩ ⟨b⟩ ⟨c⟩ h, exact @add_le_add_right _ _ _ _ b c h a }⟩
-
-instance covariant_class_add_lt : covariant_class game game (+) (<) :=
-⟨by { rintro ⟨a⟩ ⟨b⟩ ⟨c⟩ h, exact @add_lt_add_left _ _ _ _ b c h a }⟩
-
-instance covariant_class_swap_add_lt : covariant_class game game (swap (+)) (<) :=
-⟨by { rintro ⟨a⟩ ⟨b⟩ ⟨c⟩ h, exact @add_lt_add_right _ _ _ _ b c h a }⟩
```

Comment:
As a compromise, how about we replace all these instances with `example : the_typeclass := by apply_instance` with a comment saying we don't need the instance because they are inferred.